### PR TITLE
Updating DOP value range for GPS notification

### PR DESCRIPTION
**Pull request by LitvinenkoIra**
Changes for https://github.com/smartdevicelink/sdl_core/issues/2317

### CODE REVIEW

File: docs/Common/Structs/index.md [hunk 1]
```diff
@@ -197,9 +197,9 @@
 |utcMinutes|Integer|false|minvalue: 0<br>maxvalue: 59||
 |utcSeconds|Integer|false|minvalue: 0<br>maxvalue: 59||
 |compassDirection|Common.CompassDirection|false|||
-|pdop|Float|false|minvalue: 0<br>maxvalue: 10||
-|hdop|Float|false|minvalue: 0<br>maxvalue: 10||
-|vdop|Float|false|minvalue: 0<br>maxvalue: 10||
+|pdop|Float|false|minvalue: 0<br>maxvalue: 100||
```
Comment (yang1070) [inline docs/Common/Structs/index.md]: 1000, not 100

**Comment by yang1070** [comment]
@theresalech Ford has reviewed and approved this PR. It is ready for Livio to review.

**Comment by conlain-k** [comment]
07e0f8400333dd1731025ec09917e54f7f2c0d4f makes a large number of formatting changes (spaces at the end of lines), I propose reverting them and squashing into one commit on merge

**Comment by LuxoftAKutsan** [comment]
@conlain-k PR updated.

**Comment by conlain-k** [comment]
1a6855435a111a01dcf5ee1bb88f1dec4269978e seems to undo the required changes and bring back the spacing issues, but 42bfcf508531a26affb2e7eb8c0fd8709740d8ca looks perfect. What is the motivation for the second commit?

**Comment by LuxoftAKutsan** [comment]
@conlain-k will be fixed

**Comment by KhrystynaDubovyk** [comment]
@conlain-k, since [sdl_markdown_spec](https://github.com/smartdevicelink/sdl_markdown_spec#table-structure) does not have any info regarding spacing in tables 
and [the proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0175-Updating-DOP-value-range-for-GPS-notification.md) does not call for markdown text formatting in guidelines 
https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/commit/1a6855435a111a01dcf5ee1bb88f1dec4269978e may be redundant. I guess it was made just for a kind of alignment.

https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/commit/42bfcf508531a26affb2e7eb8c0fd8709740d8ca is directly related to [the proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0175-Updating-DOP-value-range-for-GPS-notification.md).

@conlain-k, do you approve keeping the changes of both commits? shall I make them the one?  
or we treat this one https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/commit/1a6855435a111a01dcf5ee1bb88f1dec4269978e as redundant?

**Comment by conlain-k** [comment]
@KhrystynaDubovyk  The formatting changes are unrelated to the proposal, so I mention them for cleanliness of the PR. 42bfcf5 does what the proposal says, but 1a68554 directly undoes these changes *and* adds whitespace changes. As the PR stands now, it doesn't actually make the required changes to the DOP values.

**Pull request merged**